tests/syrupy/extensions/amber/__snapshots__/test_amber_serializer.ambr

```diff
@@ -217,6 +217,12 @@
 # name: test_empty_snapshot.1
   ''
 # ---
+# name: test_function_in_file
+  "function_to_test(var1, var2='test_val', var3: str = 'test_val2', *, kwvar1, kwvar2='some_val') -> str"
```

Comment:
Is this going based off source code when using `inspect.signature`? I find it odd that we're getting the whitespace inconsistencies from the source code in the snapshot. It makes no functional difference and so I would expect to see it.

Reply:
Does the resulting signature string differ on different python/systems? Might just not be an issue. Though an option is removing/normalising whitespace to make it more interoperable? (like was done with  NL/CR)

Reply:
So that's actually how inspect.signature() formats the output. It puts spaces around the '=' sign if following a typehint. If I remove the spaces in the source code, it adds them back in. If I add a typehint to a different variable, it adds the spaces in.

I could strip the whitespace, but since the behavior is consistent I'm inclined to leave it. Thoughts?

Reply:
@iamogbz the result of the signature string should not differ on different systems, the output formatting is explicit. However, on different python versions it could, if they change how signature works. Does Syrupy guarantee formatting across python versions? If so, I'd need to write my own formatter.

Reply:
> Does Syrupy guarantee formatting across python versions?

We provide no explicit guarantee and I think there's an expectation that snapshots _may_ differ when you update underlying libraries/tech. That being said, we run our test suites against multiple versions of python and operating systems (though not multiple implementations) like cython etc.

Regarding whitespace, as long as it's consistent, I think it's fine.